scripts/osx/testTemplate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+OF_DIR=`pwd`;
+echo $OF_DIR;
+TEST_DIR="$OF_DIR/scripts/osx"
+cd $TEST_DIR
```

Comment:
So I have now in travis: 
``` 
    - NAME="osx" \
        SCRIPT_PATH="scripts/$NAME" \
        SCRIPT="$SCRIPT_PATH/testTemplate.sh"
    - NAME="ios" \ 
        SCRIPT_PATH="scripts/$NAME" \
        SCRIPT="$SCRIPT_PATH/testTemplate.sh"
    - NAME="docs" \
        SCRIPT_PATH="scripts/dev" \
        SCRIPT="$SCRIPT_PATH/documentation.sh"
```

This shows in the build matrix exactly where and what is being tested.
Then in each of the testTemplate scripts
```
if [ "$SCRIPT_PATH" == "" ]; then SCRIPT_PATH="."; fi
echo "Looking for iOS Template Project"
if [ ! -d "$SCRIPT_PATH/template/bin/emptyExample.app" ]; then
```